syntax = "proto3";
option objc_class_prefix = "RTG";

package Text2SpeechRPC;

service Text2SpeechService {
	rpc TextToSpeech (TTSInput) returns (TTSOutput);
}

message TTSInput {
	string text = 1;
	string lang = 2;
}

message TTSOutput {
	bytes data = 1;
}